gate-core/src/main/groovy/com/netflix/spinnaker/gate/services/internal/KeelService.java

```diff
@@ -32,7 +31,7 @@
 public interface KeelService {
 
   @GET("/resources/events/{name}")
-  List<ResourceEvent> getResourceEvents(@Path("name") String name, @Query("since") Instant since);
+  List<ResourceEvent> getResourceEvents(@Path("name") String name, @Query("limit") Integer since);
```

Comment:
the variable name should also change from `since` to `limit`

Reply:
Fixed